src/pint/models/dispersion_model.py

```diff
@@ -319,42 +319,83 @@ def __init__(self):
                 description="Dispersion measure",
             )
         )
+
+        self.add_DMX_range(None, None, dmx=0, frozen=False, index=1)
+
+        self.dm_value_funcs += [self.dmx_dm]
+        self.set_special_params(["DMX_0001", "DMXR1_0001", "DMXR2_0001"])
+        self.delay_funcs_component += [self.DMX_dispersion_delay]
+
+    def add_DMX_range(self, mjd_start, mjd_end, index=None, dmx=0, frozen=True):
+        """ Add DMX range to a dispersion model with specified start/end MJDs and DMX. 
+            returns: index [int]
+        """
+
+        #### Setting up the DMX title convention. If index is None, want to increment the current max DMX index by 1.
+        if index is None:
+            index = self.get_prefix_mapping_component("DMX_").max() + 1
+        i = f"{int(index):04d}"
+
+        if mjd_end is not None and mjd_start is not None:
```

Comment:
Do you want to allow users to add extra None/None DMX ranges?

Reply:
we needed this to allow the initial instance of DMX (the "fake" one for the model builder), which is made with None MJDs.

Reply:
Would it be better to reject None entirely and make the bogus one by hand?